examples/mysql-query-row-operation/mysql_query_row_operation.out

```diff
@@ -0,0 +1,6 @@
+# Create a Ballerina project.
+# Copy the example to the project and add relevant database driver jar details to the `Ballerina.toml` file.
```

Comment:
```suggestion
# Copy the example to the project and add the relevant database driver JAR details to the `Ballerina.toml` file.
```
Please check and correct all similar occurrences.

Reply:
Fixed in [4621ad1](https://github.com/ballerina-platform/ballerina-distribution/pull/2842/commits/4621ad1b1e00a60859c191de19de2041e0516f79)